### PR TITLE
Added Giphy script, tweaked to add nerdbot animate (me) pattern and c…

### PR DESCRIPTION
**Pull request by JustinMullin**
…onfigurable default query for when no results are found.

### CODE REVIEW

**Comment by JustinMullin** [comment]
Note: I nuked the google images script, but wasn't a good enough citizen to save the mustachify or red panda bits.  Given the state of emergency we're in without 'nerdbot animate' I figure this is worth it.  If someone else cares to disagree, feel free to amend with fixes for those features.  =)

**Comment by tvanfosson** [comment]
All the sads. Gonna miss my panda, but I can add it back later.

**Comment by JustinMullin** [comment]
Yeah, sorry @tvanfosson...the panda is only resting.

File: scripts/giphy.coffee [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  base_url: 'http://api.giphy.com/v1'
+  no_gif_found_query: process.env.HUBOT_GIPHY_DEFAULT_QUERY or 'otter'
+
+module.exports = (robot) ->
+  robot.respond /(animate|gif|giphy)( me)? (.*)/i, (msg) ->
+    giphyMe msg, msg.match[3], (url) ->
+      msg.send url
+
+giphyMe = (msg, query, cb) ->
+  endpoint = '/gifs/search'
```
Comment (mloberg) [inline scripts/giphy.coffee]: Do we want to use the [Translate endpoint](https://github.com/giphy/GiphyAPI#translate-endpoint) instead? It looks like it's designed for integrations like this.

Comment (JustinMullin) [inline scripts/giphy.coffee]: Hrm, maybe.  I'm not sure I understand what the difference is, but that does kinda sound like it's more like what we want...

**Comment by jcarlso5** [comment]
I'll merge and deploy this, we can switch api endpoints later if translate would be better.

**Comment by JaSpr** [comment]
This results in about 85% otter gifs.

**Pull request merged**